packages/server/patches/@wdio+protocols+9.0.0.patch

```diff
@@ -0,0 +1,79 @@
+diff --git a/node_modules/@wdio/protocols/README.md b/node_modules/@wdio/protocols/README.md
```

Comment:
I usually delete the README.md patches but no biggie.

Reply:
I ran into so many issues with patch-package and the yarn clean I am afraid to touch it